modules/permutiveRtdProvider.js

```diff
@@ -262,17 +262,6 @@ function getDefaultBidderFn (bidder) {
 
       return bid
     },
-    rubicon: function (bid, data, acEnabled) {
-      if (isPStandardTargetingEnabled(data, acEnabled)) {
-        const segments = pStandardTargeting(data, acEnabled)
-        deepSetValue(bid, 'params.visitor.p_standard', segments)
-      }
-      if (data.rubicon && data.rubicon.length) {
-        deepSetValue(bid, 'params.visitor.permutive', data.rubicon.map(String))
-      }
-
-      return bid
-    },
```

Comment:
This is the first special handling removed. The others are planned in follow up PRs.

Reply:
awesome ty!